### DISCUSSION TITLE
Step preview a la Digitakt

### DISCUSSION THREAD

**Discussion by n3ptun3s** [SynthstromAudible/DelugeFirmware#322] [Ideas]
Elektron served in recent updates a step preview function to the Digi machines.
Meaning that you can preview a single step in the sequencer with its parameter locks.

Maybe something like shift + triplets view + step button could play the selected step with its saved automation.

**Comment by weavermedia** [comment]
Yes, that would be really nice workflow addition! 👏 

The Triplets View button might get repurposed as a secondary shift soon so it could just be TRIPLETS+PAD

**Comment by m-m-adams** [comment]
Doesn't holding a step and pressing the audition pad do this?

**Comment by seangoodvibes** [comment]
> Doesn't holding a step and pressing the audition pad do this? 

Seems like with playback off you can press and hold a step and it will audition that square, but it only auditions the automation for the first node position in that square, it won't audition all the nodes in the square. Eg a long note with a filter sweep, you won't hear it audition the sweep.
